### DISCUSSION TITLE
YOLOv8 over YOLOv5 Performance : Automated SelfCheckout

### DISCUSSION THREAD

**Discussion by Abhijeet241093** [openvinotoolkit/openvino_notebooks#1800] [AI Solution Kits]
Hello @riacheruvu,

Suppose the GPU and storage requirements are on the server side. If you downgrade from the YOLOv8 model to the YOLOv5 model, will the performance improve?

 **Expected behavior**
 Why do we choose YOLOv8 over YOLOv5? Is it for additional flexibility or features?

**Screenshots**

 **Installation instructions (Please mark the checkbox)**
[ Yes ] I followed the installation guide at https://github.com/openvinotoolkit/openvino_notebooks#-installation-guide to install the notebooks. 

** Environment information **
Please run `python check_install.py` in the _openvino_notebooks_ directory. If the output is _NOT OK_ for any of the checks, please follow the instructions to fix that. If that does not work, or if you still encounter the issue, please paste the output of check_install.py here.
 
**Additional context**

**Comment by Abhijeet241093** [comment]
@riacheruvu

**Comment by riacheruvu** [comment]
Hi @Abhijeet241093! In this context, we are choosing YOLOv8 over YOLO5 for the performance improvements and being able to leverage the latest APIs. You could switch the model with YOLOv5 for meeting GPU/storage requirements - I will say I haven't validated this particular kit with YOLOv5 yet, so you may see different results if you try this. 

I think this conversation is more suited for [the Discussions section](https://github.com/openvinotoolkit/openvino_notebooks/discussions) - please feel free to create a new Discussions thread and we can talk further on this, if you have additional questions.

Requesting @adrianboguszewski to close this issue thread for now.

**Reply by adrianboguszewski** [reply]
Converted to discussion
